### ISSUE TITLE
The Gentleman's Discard Discharge

### ISSUE THREAD

**Issue by Dr-Kakashi** [Zamiell/hanabi-conventions#218]
- Normally, if a player performs a Gentleman's Discard and the other card is not on Finesse Position, it could be several things:
  - If the card on Finesse Position is itself playable right now, then it would be a Layered Gentleman's Discard.
  - If the card on Finesse Position is exactly one-away-from-playable, then it would be a Patch Gentleman's Discard (and call for the next-player to blind-play their Finesse Position card).
  - If the card on **First and Second** Finesse Position is two or more away from playable, or a trash card, then it is usually an emergency situation or a player is making a mistake.
- If the third option is true and it is not an emergency situation and the player is not making a mistake, then the player who is performing the weird Gentleman's Discard is instead intending for the next player to Discharge as a Gentleman Discard Discharge. Doing so will demonstrate that the card is not on the First and Second Finesse Position and is instead on slot 3.

- For example, in a 3-player game:
  - All 1's are played on the stacks.
  - Alice discards a known red 2 as a Gentleman's Discard.
  - Bob sees that Cathy has the red 2, but it is not in her Finesse Position. Cathy has a red 1 (trash) on Finesse Position, blue 4 on slot 2, followed by a red 2 on slot 3.
  - Bob knows that it would be pointless for Alice to perform a Gentleman's Discard and then for Bob to waste a clue giving a Fix Clue. So, if Alice is not making a mistake, she must be trying to communicate something deeper.
  - Bob knows that if Alice wanted him to play his Finesse Position card, this would not work, as Cathy would interpret the situation as a Patch Gentleman's Discard and would go on to misplay the red 1.
  - Bob knows that this is not the Gentleman's Discard Ejection, as red 2 is not on slot 2.
  - Bob knows that this situation signals a Gentleman's Discard Discharge, so he blind-plays his Third Finesse Position card and it is a blue 2 and successfully plays.
  - Cathy knows that some kind of Discharge is going on, since Bob blind-played his Third Finesse Position. Since the previous action was a Gentleman's Discard, this must be a Gentleman's Discard Discharge, which means that Cathy's red 2 is on her Third Finesse Position instead of her First and Second Finesse Position.
  - Cathy blind-plays the red 2.

**Comment by Zamiell** [comment]
link a game where this would be useful

**Comment by iamwhoiamhahaha** [comment]
if this is going to be approved then i'll be proposing ***The Gentleman's Discard Charm*** lol

**Comment by Dr-Kakashi** [comment]
![2020-05-25_2153](https://user-images.githubusercontent.com/48993588/82862372-508e9b80-9ed4-11ea-85fc-2fbf2e797fcd.png)

Didn't expect it to come up....I was thinking cards would be blocking, but we were actually blocked by a card in the trash pile

**Comment by iamwhoiamhahaha** [comment]
Now, let's propose *Gentleman's Discard Charm* lolol

**Comment by iamwhoiamhahaha** [comment]
just kidding

**Comment by iamwhoiamhahaha** [comment]
I don't believe zam would approve it

**Comment by Zamiell** [comment]
i dont like this convention because it causes a bomb from a mistake
e.g. when a player incorrectly assumes the identity of their card, and they discard it, now someone else bombs because its in the wrong slot

**Comment by Zamiell** [comment]
also, i don't like this convention but it is extremely edge-case and not likely to be used very often

**Comment by Zamiell** [comment]
post another screenshot of a game where it will be useful

**Comment by Zamiell** [comment]
ill leave this open for another 30 days, if no more evidence is posted then i will close this

**Comment by Zamiell** [comment]
blocked by #302

**Comment by Zamiell** [comment]
closing this in favor of 302
